content/community/practice/index.md

```diff
@@ -6,13 +6,13 @@ title: Communities of Practice
-| [Ecology](https://ecology.usegalaxy.eu/) | A web platform to get, process, analyze and visualize ecological data. | [<i class="fa fa-solid fa-graduation-cap"></i>](https://training.galaxyproject.org/training-material/topics/ecology/) [<i class="fa fa-external-link-alt" aria-hidden="true"></i>](https://ecology.usegalaxy.eu/tours/core.galaxy_ui) | 
+| [Metabolomics](https://metabolomics.usegalaxy.eu/) | Community-sdriven metabolomics Galaxy service. The main aim is to give people a public space to discover and run metabolomics tools. | [<i class="fa fa-envelope" aria-hidden="true"></i>](https://lists.galaxyproject.org/lists/metabolomics.lists.galaxyproject.org/) [<i class="fa fa-solid fa-graduation-cap"></i>](https://training.galaxyproject.org/training-material/topics/metabolomics/) |
+| [Ecology](https://ecology.usegalaxy.eu/) | A web platform to get, process, analyze and visualize ecological data. | [<i class="fa fa-solid fa-graduation-cap"></i>](https://training.galaxyproject.org/training-material/topics/ecology/) [<i class="fa fa-external-link-alt" aria-hidden="true"></i>](https://ecology.usegalaxy.eu/tours/core.galaxy_ui) |
 | [Genome Annotation](https://annotation.usegalaxy.eu)| A complete Galaxy instance for Genome Annotation. | |
 | [Climate](https://climate.usegalaxy.eu/) | The Climate Science workbench is a comprehensive set of analysis tools and consolidated workflow. | [<i class="fa fa-solid fa-graduation-cap"></i>](https://training.galaxyproject.org/training-material/topics/climate/)|
 |[Proteomics](https://proteore.org/)| Galaxy-based platform for the functional analysis and the interpretation of proteomics and transcriptomics data in biomedical research.  | [<i class="fa fa-envelope" aria-hidden="true"></i>](https://lists.galaxyproject.org/lists/galaxy-proteomics.lists.galaxyproject.org/) |
 | [Natural Language Processing](http://galaxy.lappsgrid.org/) | An open, interoperable web service platform for natural language processing (NLP) research and development. | [<i class="fa fa-envelope" aria-hidden="true"></i>](https://lists.galaxyproject.org/lists/galaxy-nlp@lists.galaxyproject.org) |
-| [MicroGalaxy](https://microbiome.usegalaxy.eu/) | All things micro* in Galaxy. The official mailing list of the microGalaxy group. |[<i class="fa fa-envelope" aria-hidden="true"></i>](https://lists.galaxyproject.org/lists/microgalaxy@lists.galaxyproject.org) |
+| [MicroGalaxy](/projects/microbial#microgalaxy-community) | Anything regarding microbial data analysis in Galaxy, with meetings, working groups, chat and mailing list |[<i class="fa fa-envelope" aria-hidden="true"></i>](https://lists.galaxyproject.org/lists/microgalaxy@lists.galaxyproject.org) |
```

Comment:
Just for my understanding: what is the reason to have the link here to the project instead of the dedicated server ?  Btw. I did not know that we got so many dedicated servers, very cool !!

Reply:
Here are listed the communities of practice so better to link to the description of the CoP than to a server